pkg/geo/geo.go

```diff
@@ -638,3 +640,17 @@ func shapeFromGeom(t geom.T) (geopb.Shape, error) {
 		return geopb.Shape_Unset, errors.Newf("unknown shape: %T", t)
 	}
 }
+
+// CompareSpatialObject compares the SpatialObject.
+// This should be consistent with the way it is stored.
```

Comment:
Can this line be more specific? Maybe `This must match the byte ordering that is be produced by encoding.EncodeGeoAscending.`